signingscript/src/signingscript/task.py

```diff
@@ -195,7 +195,7 @@ def _check_scopes_exist_and_all_have_the_same_prefix(scopes, prefixes):
 
 
 # sign {{{1
-async def sign(context, path, signing_formats):
+async def sign(context, path, signing_formats, comment=None):
```

Comment:
If we go with the above suggestion, this would be `async def sign(context, path, signing_formats, **kwargs):`